release-notes/docs/3.9.1.md

```diff
@@ -0,0 +1,15 @@
+# 3.9.1 Release Notes
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+### Bug fixes
+
+- [cht-core#6562](https://github.com/medic/cht-core/issues/6562): Muso production users have multiple task documents in non-terminal state with the same emission id
```

Comment:
Rename this issue (and then update this doc) with a more generic description so it's clear that it affects everyone not just Muso.

Reply:
Done!